src/main/java/org/apache/commons/beanutils2/BeanPredicate.java

```diff
@@ -50,6 +50,18 @@ public BeanPredicate(final String propertyName, final Predicate predicate) {
         this.propertyName = propertyName;
         this.predicate = predicate;
     }
+    
+    /**
+     * Evaluates the given object by applying the {@link #getPredicate()}
+     * to a property value named by {@link #getPropertyName()}.
+     *
+     * @param object The object being evaluated
+     * @return the result of the predicate evaluation
+     * @throws IllegalArgumentException when the property cannot be evaluated
+     */
+    public boolean evaluate(Object object) {
```

Comment:
You could probably mark this `@Deprecated`.

Reply:
No need to deprecated, we can just remove it since we are in a major release.